rotkehlchen/chain/ethereum/modules/thegraph/accountant.py

```diff
@@ -0,0 +1,79 @@
+from collections import defaultdict
+from collections.abc import Iterator
+from typing import TYPE_CHECKING
+
+from rotkehlchen.accounting.mixins.event import AccountingEventType
+from rotkehlchen.accounting.structures.evm_event import get_tx_event_type_identifier
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.ethereum.modules.thegraph.constants import CPT_THEGRAPH
+from rotkehlchen.chain.evm.accounting.interfaces import ModuleAccountantInterface
+from rotkehlchen.chain.evm.accounting.structures import TxEventSettings
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.constants import ZERO
+from rotkehlchen.fval import FVal
+
+if TYPE_CHECKING:
+    from rotkehlchen.accounting.pot import AccountingPot
+    from rotkehlchen.accounting.structures.evm_event import EvmEvent
+    from rotkehlchen.assets.asset import Asset
+    from rotkehlchen.types import ChecksumEvmAddress
+
+
+class ThegraphAccountant(ModuleAccountantInterface):
+    def reset(self) -> None:
+        self.assets_supplied: dict[tuple[ChecksumEvmAddress, Asset], FVal] = defaultdict(FVal)
```

Comment:
As discussed in our call before you don't need this here. The graph only has one asset, GRT.
```suggestion
        self.assets_supplied: dict[ChecksumEvmAddress, FVal] = defaultdict(FVal)
```